nservicebus/azure-storage-persistence/index_correlation_asp_[1,].partial.md

```diff
@@ -0,0 +1,3 @@
+NOTE: In Versions 6 and above of NServiceBus, all correlated properties are [unique by default](/nservicebus/upgrades/5to6/handlers-and-sagas.md#saga-api-changes-unique-attribute-no-longer-needed) so there is no longer a configuration setting.
```

Comment:
Since now it is broken down per-package, do we still need this `NOTE`? It seems redundant here.